exampleSite/config.toml

```diff
@@ -86,6 +86,36 @@ lastmod = ["lastmod", ":git", "date"]
   name = "Docs"
   weight = 30
 
+[[menu.main]]
+  url = "/dev"
+  name = "Dev Section"
+  weight = 35
+  identifier = "dev"
+
+[[menu.main]]
+  url = "/dev"
+  name = "Dev Section"
+  weight = 10
+  parent = "dev"
+
+[[menu.main]]
+  url = "/dev/colors"
+  name = "Colors"
+  weight = 20
+  parent = "dev"
+
+[[menu.main]]
+  url = "/dev/blog"
+  name = "Blog"
+  weight = 30
+  parent = "dev"
+
+[[menu.main]]
+  url = "/dev/events"
+  name = "Events"
+  weight = 40
+  parent = "dev"
+
```

Comment:
We aren't using config.toml in Netlify and it's not used in package.json as well. I thought since it's not in any of those files it's not used at all so updated it just to keep it in line with config-dev.toml. I removed the menus from it now.